client/pom.xml

```diff
@@ -156,5 +156,11 @@
             <version>1.10.19</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Is this intended?

Reply:
yes, for testing. Otherwise there are no logs in the console.